src/modules/tas_studio/editor/mod.rs

```diff
@@ -1229,6 +1262,75 @@ impl Editor {
                     });
                 }
             }
+
+            if let Some(hovered_line_idx) = self.hovered_line_idx {
+                // For carmera editor camera line editing.
+                if mouse.buttons.is_right_down() && !self.prev_mouse_state.buttons.is_right_down() {
+                    // Mouse was released last frame so the adjustment cannot be active.
+                    assert!(self.camera_view_adjustment.is_none());
+
+                    // There are no other adjustments in the camera editor at the moment; and anyhow
```

Comment:
I guess this comment needs an update here and in the other place

Reply:
What kind of comment change do you prefer here? Straight up deletion?

Reply:
I guess delete